openssh/README.md

```diff
@@ -0,0 +1,132 @@
+# openssh
+
+## Maintainers
+
+The Habitat Maintainers: <humans@habitat.sh>
+
+## Usage
+
+## Testing
+
+### Build the openssh core plan
+
+Enter a habitat studio
+
+```bash
+hab pkg build .
+```
+
+Exit habitat studio
+
+### Build and install the test plan
+
+Enter another habitat studio
+
+```bash
+set -x
+
+# set environment from the last openssh build
+source results/last_build.env
+
+# stop the habitat supervisor
+hab sup term
+
+# add some required software before building the openssh_test
+hab pkg install -b core/hab-studio
+hab pkg install -b core/busybox
+hab pkg install -b chef/inspec
+
+# install the openssh artifact
+hab pkg install -b results/$pkg_artifact
+
+# restart the habitat supervisor and wait for it to start
+hab sup run &
```

Comment:
I can get the tests to pass with these steps, I suggest filing a bug and doing separate investigation if the supervisor needs to be stop/started like this

```
[9][default:/src:0]# history
    1  source results/last_build.env
    2  set +x
    3  hab pkg install -b core/busybox
    4  hab pkg install -b results/$pkg_artifact
    5  hab svc load $pkg_ident
    6  hab svc status
    7  hab pkg install chef/inspec
    8  hab pkg exec chef/inspec inspec exec ./test
    9  history
[10][default:/src:0]#
```